aptos-move/aptos-vm-types/src/change_set.rs

```diff
@@ -59,26 +59,23 @@ impl GroupWrite {
     /// and ensures inner ops do not contain any metadata.
     pub fn new(
         metadata_op: WriteOp,
-        inner_ops: Vec<(StructTag, (WriteOp, Option<Arc<MoveTypeLayout>>))>,
```

Comment:
I am changing this file extensively in https://github.com/aptos-labs/aptos-core/pull/10879, can we either land this afterwards, or revert changes to change_set.rs here, and I can incorporate Vec->BTreeMap there

Reply:
I ll just rebase on top of your PR